src/serverMiddleware/routes.js

```diff
@@ -29,7 +29,8 @@ const routes = [
 	`^/nuxtversion/?$`,
 ];
 if (process.env.LERNSTORE_MODE === "EDUSHARING") {
-	routes.push(`^/content/?`);
+	// content routes except redirect (it should go to the old client, so the old content should work)
+	routes.push(`^/content/?((?!redirect).)*$`);
```

Comment:
Maybe we can do it this way?
```suggestion
	routes.push(`^/content/?$`);
	routes.push(`^/content/${uuid}/?$`);
```

```
const uuid = "[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}";
```

Reply:
ok thanks